lms/djangoapps/courseware/date_summary.py

```diff
@@ -366,18 +366,7 @@ def register_alerts(self, request, course):
         if self.date > self.current_time:
             CourseHomeMessages.register_info_message(
                 request,
-                Text(_(
-                    'If you have earned a certificate, you will be able to access it {time_remaining_string}'
-                    ' from now. You will also be able to view your certificates on your {learner_profile_link}.'
-                )).format(
-                    time_remaining_string=self.time_remaining_string,
-                    learner_profile_link=HTML(
-                        '<a href="{learner_profile_url}">{learner_profile_name}</a>'
-                    ).format(
-                        learner_profile_url=reverse('learner_profile', kwargs={'username': request.user.username}),
-                        learner_profile_name=_('Learner Profile'),
-                    ),
-                ),
+                '',  # Edraak: Message description deleted
```

Comment:
Either

```suggestion
                Text(''),  # Edraak: Message description deleted
```

or with a space

```suggestion
                Text(' '),  # Edraak: Message description deleted
```

Reply:
done

Reply:
done